src/nvim/ex_cmds.c

```diff
@@ -1258,7 +1260,7 @@ static void do_filter(
  * Call a shell to execute a command.
  * When "cmd" is NULL start an interactive shell.
  */
-void 
+void
```

Comment:
feel free to reformat signatures of functions you change to the new format (entire signature in one line if it fits in 80 characters) but otherwise please not change it.